tests/Common/ConfigurationRowTest.php

```diff
@@ -212,11 +208,18 @@ public function testConfigurationRowJsonDataTypes(callable $getClient, callable
             ]
         ];
 
+        $client->put('/v2/storage/components/wr-db/configs/main-1/rows/' . $response->id . '/state', [
```

Comment:
jinde voláš state až po updatu konfigurace, tak bych zvážil jestli to tady je vhodné dělat před tím...

Reply:
Myslim ze to je takto lepsie nie? Testuje sa tam zaroven aj to ze sa zachova ten state. Updatname state, upatnem row , skontrolujem ze ten response vratil povodny state

Reply:
Pravda, ok.